cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   containers:
   - name: fluentd-cloud-logging
-    image: gcr.io/google_containers/fluentd-gcp:1.6
+    image: kubernetes/fluentd-gcp:1.7
```

Comment:
Correct. After LGTM this should be changed to gcr.io/google_containers/fluentd-gcp:1.7
Does that make sense as a sane procedure for updating images?